src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.jsx

```diff
@@ -17,35 +16,31 @@ import SentryTypes from 'app/sentryTypes';
 import TimeSince from 'app/components/timeSince';
 import withOrganization from 'app/utils/withOrganization';
 
-const GroupTagValues = createReactClass({
-  displayName: 'GroupTagValues',
-
-  propTypes: {
+class GroupTagValues extends React.Component {
+  static propTypes = {
     api: PropTypes.object,
     group: SentryTypes.Group.isRequired,
     query: PropTypes.object,
-  },
+  };
 
-  getInitialState() {
-    return {
-      tagKey: null,
-      tagValueList: null,
-      loading: true,
-      error: false,
-      pageLinks: '',
-    };
-  },
+  state = {
+    tagKey: null,
+    tagValueList: null,
+    loading: true,
+    error: false,
+    pageLinks: '',
+  };
 
   componentWillMount() {
     this.fetchData();
-  },
+  }
 
   componentDidUpdate(prevProps) {
     const queryHasChanged = !isEqual(prevProps.query, this.props.query);
     if (queryHasChanged || prevProps.params.tagKey !== this.props.params.tagKey) {
       this.fetchData();
     }
-  },
+  }
 
   async fetchData() {
```

Comment:
Does `this` need to be bound?

Reply:
I don't think so? This method is called from inside componentDidUpdate. Does the execution scope change after the `await` resolves?

Reply:
Get's called here too https://github.com/getsentry/sentry/pull/14427/files#diff-d0c0e2ebfb90779910356bf000df25b4R94

Reply:
Missed that. Thanks :+1: